vizro-ai/docs/pages/user-guides/customize-vizro-ai.md

```diff
@@ -57,12 +59,14 @@ To ensure a deterministic answer to our queries, we've set the temperature to 0.
         llm = ChatOpenAI(
             model_name="gpt-3.5-turbo-0125",
             temperature=0,
-            max_retries=3,
+            max_retries=5,
         )
         vizro_ai = VizroAI(model=llm)
         vizro_ai.plot(df, "describe the composition of gdp in continent")
         ```
 
+In addition to customizing the model beyond its name, passing an instantiated model to `VizroAI` lets you use models that are not included in the list of [supported models](#supported-models).
```

Comment:
```suggestion
Passing an instantiated model to `VizroAI` lets you customize it, and additionally, it enables you to use an OpenAI model that is not included in the above list of [supported models](#supported-models).
```

Reply:
I added the key word "OpenAI" because I assume you cannot pass anything except that vendor's models and thought it made sense to make that explicit.